qwed_sdk/cache.py

```
@@ -257,7 +257,7 @@ def print_stats(self):
         
         try:
             from qwed_sdk.qwed_local import QWED, HAS_COLOR
-        except:
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)